src/main/scala/rocketchip/Configs.scala

```diff
@@ -260,11 +263,13 @@ class WithTestRAM extends Config(
         def addrMapEntries = Seq(
           AddrMapEntry("testram", MemSize(ramSize, MemAttr(AddrMapProt.RW))))
         def builder(
-            mmioPorts: HashMap[String, ClientUncachedTileLinkIO],
-            clientPorts: Seq[ClientUncachedTileLinkIO],
-            extra: Bundle, p: Parameters) {
+          mmioPorts: HashMap[String, ClientUncachedTileLinkIO],
+          clientPorts: Seq[ClientUncachedTileLinkIO],
+          interrupts: Seq[Bool],
+          extra: Bundle, p: Parameters) {
           val testram = Module(new TileLinkTestRAM(ramSize)(p))
           testram.io <> mmioPorts("testram")
+          interrupts.foreach(x => x := Bool(false))
```

Comment:
What I mean is, I don't see a lot of the pattern in the code base of asking some other module for its parameter value before deciding what your own IOs will be. I think @terpstra has been looking a lot at this sort of pattern, but happy to hear if you have a quick and easy way to do what you're talking about.